pkgs/tools/networking/gost/default.nix

```diff
@@ -13,8 +13,44 @@ buildGoModule rec {
 
   vendorSha256 = "1cgb957ipkiix3x0x84c77a1i8l679q3kqykm1lhb4f19x61dqjh";
 
-  # Many tests fail.
-  doCheck = false;
+  postPatch = ''
+    substituteInPlace http2_test.go \
```

Comment:
Why do we skip all of these?

Reply:
Now I'm skipping only the very specific tests that are failing. For now, it is either this or have tests disabled.